x-pack/test/case_api_integration/security_and_spaces/tests/common/cases/comments/delete_comment.ts

```diff
@@ -92,7 +92,7 @@ export default ({ getService }: FtrProviderContext): void => {
         .send()
         .expect(400);
       // make sure the failure is because of the subCaseId
-      expect(body.message).to.contain('subCaseId');
+      expect(body.message).to.contain('disabled');
```

Comment:
I removed the `subCaseId` from the message because in most cases it didn't necessarily make sense if the client contract is being called directly instead of going through the HTTP route. So I switched this to `disabled` since that's common to all the messages now.